cxplat/inc/cxplat/fault_injection.h

```diff
@@ -15,18 +16,18 @@ extern "C"
      *
      * @param[in] stack_depth Number of stack frames to capture when a fault is
      * injected.
-     * @retval STATUS_SUCCESS The operation was successful.
-     * @retval STATUS_NO_MEMORY Operation failed due to memory allocation failure.
+     * @retval 0 The operation was successful.
+     * @retval ENOMEM Operation failed due to memory allocation failure.
      */
-    NTSTATUS
-    usersim_fault_injection_initialize(size_t stack_depth) noexcept;
+    int
+    cxplat_fault_injection_initialize(size_t stack_depth) NOEXCEPT;
```

Comment:
You already `extern "c"` all these functions. Why do you have the `NOEXCEPT`?

Reply:
See https://stackoverflow.com/questions/24362616/does-the-c-standard-mandate-that-c-linkage-functions-are-noexcept

"it's a good idea that user defined extern "C" functions be also specified with noexcept, unless it is handed a pointer to C++ function as callback argument like qsort and the likes"